salt/consul/init.sls

```diff
@@ -120,7 +120,7 @@ consul:
 consul-template:
   pkg.installed: []
 
-  cmd.wait:
+  cmd.run:
```

Comment:
Salts documentation recommends moving from `cmd.wait` with `watch` to `cmd.run` with `onchanges` [here](https://docs.saltproject.io/en/latest/ref/states/all/salt.states.cmd.html#should-i-use-cmd-run-or-cmd-wait). I think we want to change the `watch` directive a few lines down to be `onchanges`